src/main/java/uk/gov/di/ipv/cri/common/library/service/AuditService.java

```diff
@@ -39,8 +49,12 @@ public void sendAuditEvent(AuditEventTypes eventType) throws SqsException {
         }
     }
 
-    private String generateMessageBody(AuditEventTypes eventType) throws JsonProcessingException {
-        AuditEvent auditEvent = new AuditEvent(Instant.now().getEpochSecond(), eventType);
+    private String generateMessageBody(String eventType) throws JsonProcessingException {
+        String fullEventType =
+                (eventPrefix == null || eventPrefix.isBlank())
+                        ? eventType
+                        : eventPrefix + "_" + eventType;
+        AuditEvent auditEvent = new AuditEvent(Instant.now().getEpochSecond(), fullEventType);
```

Comment:
add the Clock to the constructor of AuditService and consume in a similar manner to `PersonIdentityService`